### PR TITLE
Added Java support

### PR DESCRIPTION
**Pull request by iSatisfied**
I hope I did this correct, had too many issues on Windows had to move to Ubuntu

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -37,6 +38,7 @@ license_files =
     licenses/LICENSE_textmate_toml.tmbundle
     licenses/LICENSE_vscode-restructuredtext_vscode-restructuredtext
 classifiers =
+    License :: OSI Approved :: Eclipse Public License 2.0 (EPL-2.0)
```
Comment (asottile) [inline setup.cfg:41]: hmmm this shouldn't have changed, unless EPL requires relicensing and then we probably can't use this repo :(

File: bin/download-syntax [hunk 2]
```diff
@@ -262,6 +262,12 @@ REPOS = (
         license_path='LICENSE.txt',
         grammars=('syntaxes/restructuredtext.tmLanguage',),
     ),
+    Repo(
+        name='redhat-developer/vscode-java',
+        version='95ffc42',
+        license_path='LICENSE',
+        grammars=('syntaxes/java.tmLanguage.json',),
+    ),
```
Comment (asottile) [inline bin/download-syntax:270]: I think if you run the updater it'll resort this -- I can amend that in if you want

**Comment by iSatisfied** [comment]
Found a new repo with the right license and updated the grammars to resort the repos

-- forgot to hit comment..

**Review by asottile — APPROVED** [review]

**Pull request merged**